openedx/core/djangoapps/content/course_overviews/models.py

```diff
@@ -95,6 +101,8 @@ def _create_from_course(course):
             lowest_passing_grade = None
 
         return CourseOverview(
+            version=CourseOverview.VERSION,
```

Comment:
If you're use the class constructor inside this method, `_create_from_course` should be a `classmethod`, not a `staticmethod`